dev_ops/sql/Tables/CreateContractDonations.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE contract_donations (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
Can we use uuid? I think we should prefer uuids for ids and only use sequential when there is a good reason to